src/components/ImageAnalyzer.vue

```diff
@@ -0,0 +1,61 @@
+<script setup>
+import { defineProps, defineEmits } from 'vue'
+import ZoomableImage from './ZoomableImage.vue'
+
+// eslint-disable-next-line no-unused-vars
+const props = defineProps(['modelValue', 'image'])
+const emit = defineEmits(['update:modelValue'])
+
+function closeDialog() { 
+	emit('update:modelValue', false)
+}
+
+</script>
+<template>
+  <v-dialog
+    :model-value="modelValue"
+    fullscreen
+  >
+    <v-card>
+      <v-toolbar
+        density="comfortable"
+        :elevation="8"
+        :title="image.target_name"
+      >
+        <a
+          :href="image.url"
+          :download="image.basename"
+          target="_blank"
+        >
+          <v-btn
+            icon="mdi-download"
+          />
+        </a>
+        <v-btn
+          icon="mdi-close"
+          @click="closeDialog()"
+        />
+      </v-toolbar>
+      <div class="anaylzer_content">
+        <zoomable-image
+          :image-src="image.url"
+        />
+        <v-sheet
```

Comment:
this v sheet is the right hand side that contains a few details. Was created from Jon's input that we should have space on the right for some outputs